x/ccv/child/keeper/distribution.go

```diff
@@ -0,0 +1,69 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+	"github.com/cosmos/interchain-security/x/ccv/child/types"
+	// XXX delete or uncomment this block before merge
+	//ibctypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+)
+
+// Simple model, donate tokens to the fee pool of the provider validator set
+// reference: cosmos/ibc-go/modules/apps/transfer/keeper/msg_server.go
+func (k Keeper) DonateToProviderValidatorSet(ctx sdk.Context) error {
+	if !k.shouldTransmit(ctx) {
+		return nil
+	}
+
+	port := k.GetDistributionTransmissionPort(ctx)
```

Comment:
Shouldn't this just be a constant: `transfer`?

Is this to future-proof the portID changing?

Reply:
Yeah I mean we _could_ just keep it as a constant, it is also a bit more future proof to keep it in the params as I've done, but maybe that's not such a huge concern as updates to the transfer port would also likely come with hard-updates to the code base. 

I guess I also like the flow of information to be coupled, as in we get the channel from the param store, so why not also keep a reference to the port-id there? 

Kind of on the fence, I don't think it matters so much either way. Maybe we should cut it to reduce code... Alright I'm going to cut it out